doc/shell/shell_guide.md

```diff
@@ -0,0 +1,817 @@
+
+export PATH
+
+export CTP_BRANCH_NAME="develop"
+export CTP_SKIP_UPDATE=0
+```
+
+### Create a script to start consumer
+~/start_test.sh
+```
+nohup start_consumer.sh -q QUEUE_CUBRID_QA_SHELL_LINUX -exec run_shell &
+```
+Execute the script to start listening the test message after deployment. This will start a shell test when the consumer receive the test message.
+```
+cd ~
+sh start_test.sh
+```
+
+## 3.5 Deploy worker node  
+### Install CTP
+Please refer to ["CTP Installation Guide"](https://github.com/CUBRID/cubrid-testtools/blob/develop/doc/ctp_install_guide.md)
+
+### Set ~/.bash_profile
+*~/.bash_profile*
+```
+# .bash_profile
+
+# Get the aliases and functions
+if [ -f ~/.bashrc ]; then
+        . ~/.bashrc
+fi
+
+# User specific environment and startup programs
+
+PATH=$PATH:$HOME/.local/bin:$HOME/bin
+
+export CTP_HOME=$HOME/CTP
+## init_path is used when we run shell case manually on this machine.
+export init_path=$CTP_HOME/shell/init_path
+
+export PATH=$CTP_HOME/bin:$CTP_HOME/common/script:$PATH
+
+export CTP_BRANCH_NAME="develop"
+export CTP_SKIP_UPDATE=0
+
+. ~/.cubrid.sh
+export GCOV_PREFIX=/home/shell
+export GCOV_PREFIX_STRIP=2
+ulimit -c unlimited
+```
+
+### Deploy test cases
+```
+git clone --no-checkout https://github.com/CUBRID/cubrid-testcases-private-ex.git
+cd ~/cubrid-testcases-private-ex
+git config core.sparseCheckout true
+echo 'shell/*' > ~/cubrid-testcases-private-ex/.git/info/sparse-checkout
+git checkout develop
+```
+
+### Make directories for test
+```
+cd
+mkdir do_not_delete_core
+mkdir ERROR_BACKUP
+```
+
+### Create .cubrid.sh file 
+If cubrid has never been installed on the machine, we need create file '.cubrid.sh' at $HOME path manually.  
+*.cubrid.sh file:*    
+```
+CUBRID=/home/shell/CUBRID
+CUBRID_DATABASES=$CUBRID/databases
+if [ "x${LD_LIBRARY_PATH}x" = xx ]; then
+  LD_LIBRARY_PATH=$CUBRID/lib
+else
+  LD_LIBRARY_PATH=$CUBRID/lib:$LD_LIBRARY_PATH
+fi
+SHLIB_PATH=$LD_LIBRARY_PATH
+LIBPATH=$LD_LIBRARY_PATH
+PATH=$CUBRID/bin:$PATH
+export CUBRID
+export CUBRID_DATABASES
+export LD_LIBRARY_PATH
+export SHLIB_PATH
+export LIBPATH
+export PATH
+```
+
+# 4 Regression Test
+We execute shell test for each CI build, and execute code coverage test monthly.  
+Both of the test are started automatically when the controller receive a test message. We just need to verify the test results, and report the issues.
+
+## 4.1 Verify regression test results
+### Result overview
+Open qahome in browser, then select build number. 'Function Result' page will be displayed.
+Find shell and shell_debug in linux part, and find shell in windows part.  
+![shell_linux](./image1.png)
+![shell_windows](./image2.png)
+If there are crash failures, an alarm icon will appear in the 'Total' column, like this:  
+![shell_crash](./image12.png)  
+In the overview page, we should check the values in all the colomns. The most important check items are:  
+1. Whether the test is completed ('Test Rate' column)

```

Comment:
Convert to assertion:

Test Rate should be 100%
Verified Rate should be 100%

For elapse time, it should go to "Low Performance Scenarios" page to check.